webapp/src/Service/ScoreboardService.php

```diff
@@ -437,32 +439,49 @@ public function calculateScoreRow(
             //   - or already judged to be correct (if it is judged but not correct,
             //     it is not a first to solve)
             // - or the submission is still queued for judgement (judgehost is NULL).
-            $verificationRequiredExtra = $verificationRequired ? 'OR j.verified = 0' : '';
+            // If there are no valid correct submissions submitted earlier but there
+            // are submissions awaiting judgement, we are potentially the first to solve.
+            // We need to keep track of this since we later need to set the actual first to solve.
+            $verificationRequiredExtra = ($verificationRequired && !$useExternalJudgements) ? 'OR j.verified = 0' : '';
```

Comment:
Why is this change needed?

Reply:
External judgements don't have a verified status in this sense. This just never worked.